src/cmd/build.js

```diff
@@ -131,7 +131,8 @@ async function defaultPackageCreator({
   await eventToPromise(stream, 'close');
 
   if (showReadyMessage) {
-    log.info(`Your web extension is ready: ${extensionPath}`);
```

Comment:
I think the old message was correct. Here is why:
* A built `.zip` runs fine in Chrome
* A built `.zip` can be submitted for signing by Mozilla through the [API](http://addons-server.readthedocs.io/en/latest/topics/api/signing.html).
* Submitting the `.zip` to addons.mozilla.org is only one of many things you can do
* The bug you noticed where Firefox thinks the extension is corrupt is a [bug in Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1224591#c6). In other words, the error message is terrible but no one is convinced enough to fix (it should be fixed IMO).